app.rb

```diff
@@ -1,3 +1,4 @@
+# -*- encoding : utf-8 -*-
```

Comment:
Hey @yamilethmedina - so I'm reading that Ruby 2.0, makes UTF-8 the default encoding, removing the need for magic comments. Since the `Gemfile` now has it version at 2.2.4, would you be okay with removing the magic comments and sanity testing the user tests again?